lib/BloqadeLattices/src/BloqadeLattices.jl

```diff
@@ -38,10 +38,15 @@ export # types
     # visualize
     img_atoms,
     img_maskedgrid,
-    ByDensity
+    ByDensity,
+    # bounds
+    Parallelepiped,
+    Tile,
+    within_cell
 
 include("lattice.jl")
 include("neighbors.jl")
 include("visualize.jl")
+include("tiles.jl")
```

Comment:
```suggestion
include("supercell.jl")
```

rename tiles.jl to supercell.jl